nominal/core/client.py

```diff
@@ -442,6 +444,48 @@ def get_commensurable_units(self, unit_symbol: str) -> Sequence[scout_units_api.
         """Get the list of units that are commensurable (convertible to/from) the given unit symbol"""
         return self._clients.units.get_commensurable_units(self._clients.auth_header, unit_symbol)
 
+    def get_channel(self, rid: str) -> Channel:
+        """Get metadata for a given logical series by looking up its rid
+        Args:
+            rid: Identifier for the logical series to look up
+        Returns:
+            Resolved metadata for the requested logical series
+        Raises:
+            conjure_python_client.ConjureHTTPError: An error occurred while looking up the logical series.
```

Comment:
not for now, just food for thought for later - we should probably consider wrapping all these conjure errors